spaceship/lib/spaceship.rb

```diff
@@ -2,6 +2,7 @@
 require 'spaceship/base'
 require 'spaceship/client'
 require 'spaceship/launcher'
+require 'fastlane_core'
```

Comment:
@KrauseFx  is UI.message available in spaceship since the contributing guide says to use that?